### PR TITLE
November planning meeting notes

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: meeting_notes/20161112_planning-meeting-notes.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+- _PARKED_ figure out whether we need some legal status and how to acquire it
+
+## Current Decisions
+
+### Logo
+
+- We have [placeholder logo](https://tomesh.net/images/tomesh-icon.svg)) designed by garry already on website, chat, and github account
+- William designed a few versions of a [new logo](https://matrix.tomesh.net/_matrix/media/v1/download/tomesh.net/yszGAgAETuPkMkzMUZnoMchI)
+- Discussed whether to adopt the new logo, or use the old logo permanently **could not consense on either**
+  - concerns about appearance of new logo, lack of clarity as a 'Toronto' symbol, consistent brand identity
```
Comment (benhylau) [inline meeting_notes/20161112_planning-meeting-notes.md:97]: > lack of clarity as a 'Toronto' symbol

Wasn't this a concern with the existing logo? The new one is clearly Toronto :P

Comment (dcwalk) [reply meeting_notes/20161112_planning-meeting-notes.md:97]: oh I thought anastasia brought up that she didn't realize it was the CN tower

Comment (benhylau) [reply meeting_notes/20161112_planning-meeting-notes.md:97]: Ahh I must've missed that, parallel conversations going on my side of the table! Looks good to merge then :)

Comment (dcwalk) [reply meeting_notes/20161112_planning-meeting-notes.md:97]: thanks!

**Pull request merged**